mlflow/pyfunc/__init__.py

```diff
@@ -724,6 +727,85 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl)
 
 
+def _download_model_conda_env(model_uri):
+    conda_yml_file_name = _get_flavor_configuration_from_uri(model_uri, FLAVOR_NAME)[ENV]
+    return _download_artifact_from_uri(os.path.join(model_uri, conda_yml_file_name))
+
+
+def get_model_dependencies(model_uri, format="pip"):  # pylint: disable=redefined-builtin
+    """
+    Given a model URL, and format ("pip" or "conda", default value "pip"),
+    return the downloaded dependency file path.
+    If "pip" format specified but model does not have "requirements.txt" file,
+    fallback to parse the pip section of the model's "conda.yaml" and ignore other
+    non-pip dependencies in the "conda.yaml" file.
+    """
+    req_file_uri = os.path.join(model_uri, _REQUIREMENTS_FILE_NAME)
```

Comment:
```suggestion
    req_file_uri = os.path.join(model_uri, _REQUIREMENTS_FILE_NAME)
```

I think this causes an issue in Windows.